styleguide/_themes/derek/scss/patterns/grid/businessResource.scss

```diff
@@ -1,14 +1,25 @@
-.resource {
+$hover-color: 12px solid rgba(0, 0, 0, .4);
+
+.businessResource-widget {
+  @include make-md-column(12);
   background-color: $white;
   border-top: 12px solid $blue-steel;
-  margin: 10px;
   overflow: auto;
-  padding: 20px 20px 5px;
+  padding: 20px 30px 5px;
   position: relative;
+}
+
+
+.businessResource-widget-hover {
+  color: $gray-darker;
+  font-weight: 600;
+  padding-bottom: 20px;
+  text-decoration: none;
```

Comment:
I think the `color`, `font-weight`, `padding-bottom` and `text-decoration` here should go under `.businessResource-widget` instead, so we're only dealing with the hover state here.